app.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "kepler",
+  "buildpacks": [{
+    "url": "https://github.com/dulaccc/heroku-buildpack-geodjango.git#1.1"
```

Comment:
I do see reference to choosing a buildpack using a config setting here if you were concerned this might not actually be doing it's thing:
https://devcenter.heroku.com/articles/buildpacks